src/app/pipes/ticket-option.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { MaybeNull } from '../types/maybe-null';
+import { TicketOption } from '../enums/ticket-option';
+import { TicketOptionItems } from '../components/wizard/ticket-options/ticket-option-items';
+import { ISelectItem } from '../interfaces/select-item.interface';
+
+@Pipe({
+  name: 'ticketOption',
+  standalone: true
+})
+export class TicketOptionPipe implements PipeTransform {
+
+  transform(value: MaybeNull<TicketOption>): string {
+    const foundItem = TicketOptionItems.find(
+      (item: ISelectItem<MaybeNull<TicketOption>>): boolean => item.value === value
```

Comment:
`ISelectItem` ist parametrisierbar um den type des values mitzugeben. Die funktion hier basiert auf `===`, was nur mit rudimentären types wie string und number funktioniert.

Das könnte man z.B. gut in einem Unit test prüfen, falls sich der hinterliegende type von `TicketOption` mal ändert.